docs/rooms/fireRoom.md

```diff
@@ -0,0 +1,59 @@
+# Fire Room
+
+A room with three different paths.
+One path leads to a person, the second path leads to a item and the third path leads no where.
+Each path is on fire but the player is given two buckets of water.
+The goal of this room is to save the person.
+
+## PlayerStates
+
+- **AtEntrance**: The player is in the main room with x buckets of water.
```

Comment:
Per the [new guidelines](https://github.com/normal-devs/klicker-knight/blob/b2e0c64bf9ab5cf02c852806e7528ec75e13b8a0/README.md?plain=1#L55-L56) can you update these to be in the same perspective as the [other docs](https://github.com/normal-devs/klicker-knight/blob/b2e0c64bf9ab5cf02c852806e7528ec75e13b8a0/docs/rooms/exampleRoom3Room.md?plain=1#L11-L13)? This way the implementation closely matches the docs, and it's easier to visualize the output when designing the room.

Reply:
updated